api/hub/src/resolvers/query.ts

```diff
@@ -227,6 +236,77 @@ const query = {
     const interests: string[] = await dataSources.profileAPI.getInterests(pubKey);
     return Array.from(interests);
   },
+  getIntegrationInfo: async (_source, { integrationIDs }, { dataSources }) => {
+    const results = [];
+    const queryPrefix = '0xf-';
+    for (const integrationID of integrationIDs) {
+      const cacheKey = `${queryPrefix}${integrationID}`;
+      const hasKey = await registryCache.has(cacheKey);
+      if (hasKey) {
+        results.push(registryCache.get(cacheKey));
+        continue;
+      }
+      const data = await getIcRegistryContract().getPackageInfo(integrationID);
+      if (data.author === ethers.constants.AddressZero) {
+        results.push({
+          id: integrationID,
+        });
+        continue;
+      }
+      const integrationInfo = {
+        id: integrationID,
+        name: data.integrationName,
+        author: data.author,
+        latestReleaseId: data.latestReleaseId,
+        integrationType: data.integrationType,
+        enabled: data.enabled,
+      };
+      results.push(integrationInfo);
+      await registryCache.set(cacheKey, integrationInfo, 3600);
+    }
+    return results;
+  },
+  getLatestRelease: async (_source, { integrationIDs }, { dataSources }) => {
+    const results = [];
+    const queryPrefix = '0xfi-';
```

Comment:
why is this queryPrefix different from the one in getIntegrationInfo (0xf- vs 0xfi-)?